collections/videosCollection.js

```diff
@@ -0,0 +1,53 @@
+// api key is placed on top, instead of making multiple copies
+// also easier to replace or delete, incase something goes wrong.
+var youtubeAPIkey = 'AIzaSyD7G4ceeUxcKA-EHzzdmNGT_gvoO7YnsvY';
+
+// a fetch request variable, so that it can be customized or reused by different function for different request.
+// The request is set to filtered to creative commons, so licensed videos will not show up on reponses.
+var defaultGetRequest = 'https://www.googleapis.com/youtube/v3/search?maxResults=10&part=snippet&type=video&videoLicense=creativeCommon&key=' + youtubeAPIkey;
```

Comment:
Max results was there for the ability to scroll. By default, it returns 5, but 5 videos in the container makes it 'unscrollable', unless the container is smaller than 5, since it has to be overflow.